front/src/modules/scenario/components/ScenarioExplorer/ScenarioExplorer.tsx

```diff
@@ -110,7 +131,7 @@ const ScenarioExplorer = ({
       globalStudyId &&
       studyDetails &&
       globalScenarioId &&
-      scenarioDetails ? (
+      (scenarioDetails || scenarioV2Details) ? (
```

Comment:
Instead of testing everywhere whether you are in V1 or V2, you can create a new const like this:
```
  const scenario = useMemo(
    () => (trainScheduleV2Activated ? scenarioV2 : scenarioV1),
    [trainScheduleV2Activated, scenarioV1, scenarioV2]
  );
```

and then you can use scenario everywhere